### PR TITLE
Enums with constants

### PR DESCRIPTION
**Pull request by Futsch1**
I have realized that it was not possible to initialize enumerators with constant values defined in the ASN specification, only literals were allowed.

Now, constants work as well.

### CODE REVIEW

File: asn1tools/codecs/compiler.py [hunk 1]
```diff
@@ -881,6 +881,20 @@ def get_size_range(self, type_descriptor, module_name):
 
         return minimum, maximum, has_extension_marker
 
+    def get_enum_values(self, type_descriptor, module_name):
+        """Converts the enum values to ints. Direct conversion if they are already
+        integers, value lookup in case they are constants.
+
+        """
+        enum_values = []
```
Comment (eerimoq) [inline asn1tools/codecs/compiler.py:889]: Add a blank line before this line.

File: asn1tools/codecs/compiler.py [hunk 2]
```diff
@@ -881,6 +881,20 @@ def get_size_range(self, type_descriptor, module_name):
...
+        for value in type_descriptor['values']:
```
Comment (eerimoq) [inline asn1tools/codecs/compiler.py:890]: Add a blank line before this line.

File: asn1tools/codecs/compiler.py [hunk 3]
```diff
@@ -881,6 +881,20 @@ def get_size_range(self, type_descriptor, module_name):
...
+            if value != EXTENSION_MARKER and not isinstance(value[1], int):
+                lookup = self.lookup_value(value[1], module_name)
+                enum_values.append((value[0], lookup[0]['value']))
+            else:
+                enum_values.append(value)
+        return enum_values
```
Comment (eerimoq) [inline asn1tools/codecs/compiler.py:896]: Add a blank line before this line.

File: tests/test_ber.py [hunk 4]
```diff
@@ -402,9 +402,9 @@ def test_enumerated(self):
             ('E', 'd', b'\x0a\x01\x03'),
             ('E', 'f', b'\x0a\x01\x04'),
             ('E', 'g', b'\x0a\x01\x05'),
-            ('E', 'e', b'\x0a\x01\x19')
+            ('E', 'e', b'\x0a\x01\x19'),
+            ('H', 'a', b'\x0a\x01\x0a')
         ]
-
```
Comment (eerimoq) [inline tests/test_ber.py:407]: Do not remove this blank line.

File: asn1tools/codecs/compiler.py (hunk 1 above)
Comment (Futsch1) [reply asn1tools/codecs/compiler.py:889]: Done

File: asn1tools/codecs/compiler.py (hunk 2 above)
Comment (Futsch1) [reply asn1tools/codecs/compiler.py:891]: Done

File: asn1tools/codecs/compiler.py (hunk 3 above)
Comment (Futsch1) [reply asn1tools/codecs/compiler.py:898]: Done

File: tests/test_ber.py (hunk 4 above)
Comment (Futsch1) [reply tests/test_ber.py:407]: Done

**Pull request merged**